crates/api_crud/src/comment/create.rs

```diff
@@ -51,8 +51,17 @@ pub async fn create_comment(
 
   // Check for a community ban
   let post_id = data.post_id;
-  let post = get_post(post_id, &mut context.pool()).await?;
-  let community_id = post.community_id;
+  let post_view = PostView::read(
```

Comment:
Reading the full post_view here instead of just the post, to be able to get the `post_view.counts.comments` below.

Reply:
Better to put this kind of thing directly in a code comment.